kinode/packages/app_store/ui/src/App.tsx

```diff
@@ -63,6 +63,7 @@ function App() {
 
   useEffect(() => {
     provider?.getNetwork().then(network => {
+      // TODO: don't have this fire 20 times on load
```

Comment:
I really really wish we could fix this for 0.6.3

Reply:
yeah if you want me on that, I can make another branch and try to get it done for 0.6.3